packages/bvaughn-architecture-demo/components/console/CurrentTimeIndicator.module.css

```diff
@@ -2,4 +2,6 @@
   width: 100%;
   height: 2px;
   background-color: var(--color-current);
+  position: absolute;
+  z-index: 5;
```

Comment:
This caused a visual regression:
![Screen Shot 2022-09-16 at 2 36 21 PM](https://user-images.githubusercontent.com/29597/190708456-55f79d01-3e44-4441-8ee3-d275d7a0f40c.png)

Any idea why it was necessary? Looks like if I remove it, it fixes the regression (and doesn't seem to obviously break anything)

Reply:
Removed in #7758